walkthroughs/howto-k8s-alb/README.md

```diff
@@ -33,11 +33,12 @@ Note: Only [deploy the ALB Ingress Controller](https://kubernetes-sigs.github.io
 
 Check the events of the ingress to see what has occur.
 
-    ```bash
+    ```
```

Comment:
interest that putting the language broke the rendering here, but that is what it looks like...

Reply:
Looking at the rendered file in master it looks like there is still the issue. Could be an issue with line 28 which has a `.` after the backticks

Reply:
Oh my bad @dfezzie

I hadn't looked at the rendered markdown on @realvz 's fork to verify. To be fair it actually looks mostly* correct with this change but yeah its still bugged. :)